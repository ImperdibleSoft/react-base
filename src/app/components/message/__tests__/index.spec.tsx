import * as React from 'react';
import { shallow } from 'enzyme';

import Message from '..';

describe('Message', () => {
  it('should render the default message', () => {
    const component = shallow(
      <Message>
        <p>Test</p>
      </Message>
    );
    expect(component).toMatchSnapshot();
  });

  it('should render the error message', () => {
    const component = shallow(
      <Message type="error">
        <p>Test error</p>
      </Message>
    );
    expect(component).toMatchSnapshot();
  });

  it('should render the success message', () => {
    const component = shallow(
      <Message type="success">
        <p>Test success</p>
      </Message>
    );
    expect(component).toMatchSnapshot();
  });

  it('should render the warning message', () => {
    const component = shallow(
      <Message type="warning">
        <p>Test warning</p>
      </Message>
    );
    expect(component).toMatchSnapshot();
  });
});
