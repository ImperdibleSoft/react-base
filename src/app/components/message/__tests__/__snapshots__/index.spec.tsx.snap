// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Message should render the default message 1`] = `
<Wrapper
  backgroundColor="#d6e3ec"
  color="#273238"
>
  <p>
    Test
  </p>
</Wrapper>
`;

exports[`Message should render the error message 1`] = `
<Wrapper
  backgroundColor="#f3b5b4"
  color="#640000"
>
  <p>
    Test error
  </p>
</Wrapper>
`;

exports[`Message should render the success message 1`] = `
<Wrapper
  backgroundColor="#c1fdb5"
  color="#006400"
>
  <p>
    Test success
  </p>
</Wrapper>
`;

exports[`Message should render the warning message 1`] = `
<Wrapper
  backgroundColor="#fdfeb5"
  color="#646400"
>
  <p>
    Test warning
  </p>
</Wrapper>
`;
